Index: pixman/configure.ac
===================================================================
--- pixman.orig/configure.ac	2007-09-19 23:20:49.000000000 +0200
+++ pixman/configure.ac	2007-09-19 23:20:51.000000000 +0200
@@ -88,7 +88,7 @@
 dnl ===========================================================================
 dnl Check for MMX
 
-MMX_CFLAGS="-mmmx -msse -Winline --param inline-unit-growth=10000 --param large-function-growth=10000"
+MMX_CFLAGS="-mmmx -Winline --param inline-unit-growth=10000 --param large-function-growth=10000"
 
 have_mmx_intrinsics=no
 AC_MSG_CHECKING(For MMX/SSE intrinsics in the compiler)
